Cargo.toml

```
@@ -25,3 +25,4 @@ pyo3 = { version = "0.22.3", features = [
     "abi3-py38",
     "py-clone",
 ] }
+pyo3-file = "0.9.0"
```

Trailing newline here, please 🙂 

```suggestion
pyo3-file = "0.9.0"

```